tests/benchmarks/multicore/idle_twim/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+cmake_minimum_required(VERSION 3.20.0)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+if (NOT SYSBUILD)
+	message(WARNING
```

Comment:
FATAL_ERROR if sysbuild is required

Reply:
Why, existing samples in benchmarks have all the same configuration with 'warning' ?

Reply:
If it needs sysbuild to work, then fatal error, if it does not, then warning. Also don't mark conversations as resolved without either implementing the fix or the other person dismissing their own comments, thanks.